lib/wallaby/experimental/chrome.ex

```diff
@@ -0,0 +1,161 @@
+defmodule Wallaby.Experimental.Chrome do
+  @behaviour Wallaby.Driver
+
+  @pool_name Wallaby.ChromedriverPool
+
+  alias Wallaby.Session
+  alias Wallaby.Experimental.Chrome.Webdriver
+  # alias Wallaby.Phantom.Driver
+  alias Wallaby.Experimental.Selenium.WebdriverClient
+
+  def child_spec(), do: :poolboy.child_spec(@pool_name, poolboy_config(), [])
+
+
+  def start_session(opts \\ []) do
+    chromedriver = :poolboy.checkout(@pool_name, true, :infinity)
+    start_session(chromedriver, opts)
+  end
+
+  def start_session(chromedriver, opts) do
+    {:ok, base_url} = Wallaby.Experimental.Chrome.Chromedriver.base_url(chromedriver)
+    capabilities = Keyword.get(opts, :capabilities, %{})
+    create_session_fn = Keyword.get(opts, :create_session_fn,
+                                    &Webdriver.create_session/2)
+
+    capabilities = Map.merge(default_capabilities(), capabilities)
+
+    with {:ok, response} <- create_session_fn.(base_url, capabilities) do
```

Comment:
is there any reason to use `with` when we just have one clause? i.e. why not just write it wthout `with`?